index.html

```diff
@@ -2,9 +2,14 @@
 <html lang="en">
   <head>
     <meta charset="UTF-8" />
-    <link rel="icon" type="image/svg+xml" href="/src/public/favicon.svg" />
     <meta name="viewport" content="width=device-width, initial-scale=1.0" />
-    <title>Vite App</title>
+    <title>King and pigs</title>
```

Comment:
```suggestion
    <title>King and Pigs</title>
```

Dałbym tu duże "P", pasuje bardziej do nagłówka strony głównej.

Reply:
słusznie !